.github/workflows/ci.yml

```diff
@@ -0,0 +1,150 @@
+name: CI
+on:
```

Comment:
Can we please add an empty line between elements on the same level, that do not belong to each other?

```suggestion

on:
```

Reply:
fixed with d07f7d6 __ALL OCCURRENCES__

Reply:
Thanks a lot!